_includes/manuals/3.13/3.2.2_installer_options.md

```diff
@@ -0,0 +1,2547 @@
+
+The installer is a collection of Puppet modules, which have a large number of parameters available to customize the configuration.  Parameters can be set by running `foreman-installer` with arguments, e.g. `--foreman-initial-admin-password`, changing settings in interactive mode or by setting up an answers file.
+
+The precedence for settings is for those set by arguments to foreman-installer or interactive mode, then the answers file, then the Puppet manifest defaults.
+
+#### foreman-installer arguments
+
+Every parameter available in the installer can be set using command line arguments to `foreman-installer`.  Run `foreman-installer --help` for most options, or `foreman-installer --full-help` for a list of every available option.
+
+When running the installer, all arguments passed on the command line will be persisted by default to `/etc/foreman-installer/scenarios.d/foreman-answers.yaml` and used automatically on subsequent runs, without needing to specify those arguments again.  This persistence can be disabled with the `-b` option.
+
+#### Interactive mode
+
+The installer also provides a text driven interface to customize configuration parameters, and can be run by executing:
+
+{% highlight bash %}
+foreman-installer -i
+{% endhighlight %}
+
+#### Plugins and compute resources
+
+The installer contains a number of high level modules (e.g. "foreman", "puppet") and additionally a number of smaller modules for additional functionality, such as plugins and compute resource support.  These can be added with the "--enable" switches, or the default options can be disabled with "--no-enable" switches.
+
+More information about compute resources can be found in the [Compute Resources section](/manuals/{{page.version}}/index.html#5.2ComputeResources) and plugins in the [Plugins section](/manuals/{{page.version}}/index.html#6.Plugins).
+
+<!-- Generated based on `foreman-installer -h | grep enable-` -->
+<div>
```

Comment:
Please make sure this is equal to nightly

Reply:
The script didn't do this... I had to manually create these files to match nightly